package.json

```diff
@@ -37,6 +37,8 @@
     "@rollup/plugin-json": "^5.0.1",
     "@rollup/plugin-node-resolve": "^15.0.1",
     "@rollup/plugin-terser": "^0.1.0",
+    "@rollup/plugin-typescript": "^9.0.2",
```

Comment:
Also missing `tslib`

Reply:
made the suggested change as part of commit 09be8e5fdff162327be808f6336dcd762cfd9d73